packages/playground/src/components/app-home/app-home/app-home.tsx

```diff
@@ -28,28 +30,56 @@ export class AppHome {
   }
 
   render() {
+    let content;
+
+    if (!this.hasDetectedNetwork) {
+      content = (
+        <div class="loading">
```

Comment:
@snario Why not use `<widget-spinner />`? We already have a component for this.

Reply:
I wanted the specific effect of pulsating circles instead of a spinning circle. Maybe I should make that a flavour / mode of that widget?

Reply:
Oh, cool. :+1: We can add that as a setting later to the component. Didn't notice it was a different shape.